solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Features/Details/Scenes/LaunchDetails/Views/LaunchDetailsBodyView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  LaunchDetailsBodyView.swift
+//  SpaceApp
+//
+//  Created by Razee Hussein-Jamal on 08/03/2022.
+//
+
+import SwiftUI
+
+struct LaunchDetailsBodyView: View {
+    let title: String
+    let description: String
+    let buttonTitle: String
```

Comment:
**[CHANGE REQUEST]**
Consider encapsulating this in a model object like: 
```suggestion
    struct Model {
        let title: String
        let description: String
        let buttonTitle: String
     }
     let model: Model
     let buttonAction: () -> Void // leave the action outside of the model, since it's not related to data.
```

Also, the action is missing